### PR TITLE
DOC: add literature reference for apply_inverse_cov

### PR DESCRIPTION
**Pull request by dengemann**
This PR adds a reference for the recently introduced `apply_inverse_cov`function.

cc @agramfort

### CODE REVIEW

File: examples/inverse/plot_mne_cov_power.py [hunk 1]
```diff
@@ -5,7 +5,16 @@
...
+estimating the source timecourses and then computing their power. This
+approach has been originallyd developed to correct for individual field spread
+using source localization in the context of predictive modeling [1]_.
+
+References
+----------
+.. [1] Sabbagh, D., Ablin, P., Varoquaux, G., Gramfort, A., & Engemann, DA.
+       (2020). Predictive regression modeling with MEG/EEG: from source
+       power to signals and cognitive states. NeuroImage.
+       https://doi.org/10.1016/j.neuroimage.2020.116893
```
Comment (larsoner) [inline examples/inverse/plot_mne_cov_power.py:17]: You should use `.. footbibliography` and add to `doc/references.bib`

File: examples/inverse/plot_mne_cov_power.py [hunk 2]
```diff
@@ -5,7 +5,16 @@
 
 We can apply the MNE inverse operator to a covariance matrix to obtain
 an estimate of source power. This is computationally more efficient than first
-estimating the source timecourses and then computing their power.
+estimating the source timecourses and then computing their power. This
+approach has been originallyd developed to correct for individual field spread
+using source localization in the context of predictive modeling [1]_.
```
Comment (larsoner) [inline examples/inverse/plot_mne_cov_power.py:10]: `:footcite:`

File: doc/references.bib [hunk 3]
```diff
@@ -1324,6 +1324,15 @@ @article{RousseletEtAl2010
   year = {2010}
 }
 
+
+@article{Sabbagh2020,
+  author = {Sabbagh, D. and Ablin, P. and Varoquaux, G. and Gramfort, A. and Engemann, DA.},
+  title = {Predictive regression modeling with MEG/EEG: from source power to signals and cognitive states},
+  journal = {NeuroImage},
+  year = {2020},
+  doi = {https://doi.org/}10.1016/j.neuroimage.2020.116893}
```
Comment (drammock) [inline doc/references.bib]:
```suggestion
  doi = {https://doi.org/10.1016/j.neuroimage.2020.116893}
```

**Comment by dengemann** [comment]
@larsoner I saw that some in some other places footcite has not been used either. While I'm at it shall I fix it in other places?

Also @larsoner @agramfort while this ref gives credit to the Sabbagh paper whose original research code went into the initial contribution, probably the idea of apply MNE operators to covariances may be older. If you are aware of other references that would deserve credit here I'd be very happy to add them here.

File: doc/references.bib [hunk 4]
```diff
@@ -1324,6 +1324,15 @@ @article{RousseletEtAl2010
   year = {2010}
 }
 
+
+@article{Sabbagh2020,
+  author = {Sabbagh, D. and Ablin, P. and Varoquaux, G. and Gramfort, A. and Engemann, DA.},
```
Comment (drammock) [inline doc/references.bib:1329]:
```suggestion
  author = {Sabbagh, David and Ablin, Pierre and Varoquaux, Gaël and Gramfort, Alexandre and Engemann, Denis A.},
```

File: examples/inverse/plot_mne_cov_power.py [hunk 5]
```diff
@@ -6,15 +6,14 @@
...
+predictive modeling.
 
 References
 ----------
-.. [1] Sabbagh, D., Ablin, P., Varoquaux, G., Gramfort, A., & Engemann, DA.
-       (2020). Predictive regression modeling with MEG/EEG: from source
-       power to signals and cognitive states. NeuroImage.
-       https://doi.org/10.1016/j.neuroimage.2020.116893
+.. footbibliography::
+
```
Comment (drammock) [inline examples/inverse/plot_mne_cov_power.py:16]:
```suggestion
```

Comment (drammock) [reply examples/inverse/plot_mne_cov_power.py:16]: this blank line after footbibliography will cause a docstring style error

**Review by drammock — APPROVED** [review]
LGTM, besides my suggestions.

File: doc/references.bib (hunk 3 above)
Comment (drammock) [reply doc/references.bib]: i see you already fixed this one.

**Comment by larsoner** [comment]
> @larsoner I saw that some in some other places footcite has not been used either. While I'm at it shall I fix it in other places?

There are a lot of places it hasn't been updated, in general I update them in whatever function/file I happen to be working on at a given time.

**Comment by dengemann** [comment]
> > @larsoner I saw that some in some other places footcite has not been used either. While I'm at it shall I fix it in other places?
> 
> 
> 
> There are a lot of places it hasn't been updated, in general I update them in whatever function/file I happen to be working on at a given time.

Ok I'll fix a few I saw jn the cov documentation

**Comment by dengemann** [comment]
@larsoner @larsoner I have addressed your concerns and fixed a few more footcite cases on the fly.

I also dropped an e-mail to John Mosher to see if there is any important paper to be cited for the general idea of projecting the data cov, beyond this implementation that comes from our work with @DavidSabbagh

**Comment by dengemann** [comment]
I think we do not necessarily have to wait for a reply from John Mosher.

Once we have more info, we can amend.

If you do not have other concerns @larsoner @drammock I'm happy to merge in principle.

I see some CI's have been failing. Do you think this is related to the changes proposed?

**Comment by drammock** [comment]
Looks like one Travis failure was a timeout when trying to download conda, and the other was a "total time exceeded" error which is probably not due to this PR. But the CircleCI failure is relevant:

`/home/circleci/project/mne/cov.py:docstring of mne.compute_covariance:43: WARNING: Unknown target name: "1".`

**Comment by dengemann** [comment]
I have not heard from John Mosher yet. I suggest we move on and improve refs once we have new insights. Good to go from my end @larsoner @drammock

**Comment by larsoner** [comment]
See @drammock's comment above:

> /home/circleci/project/mne/cov.py:docstring of mne.compute_covariance:43: WARNING: Unknown target name: "1".

I think it's up on line 722

**Comment by dengemann** [comment]
> I think it's up on line 722

Oopsy. I did not have not fully digested the implications. Let me investigate.

**Comment by dengemann** [comment]
@larsoner @drammock squashed commits and (hopefully) fixed the error.

**Comment by larsoner** [comment]
Thanks @dengemann !

**Pull request merged**